### PR TITLE
app-admin/passwordsafe: version bump to 1.09.0

### PR DESCRIPTION
**Pull request by madmartin**
Closes: https://bugs.gentoo.org/686486
Closes: https://bugs.gentoo.org/696598
Package-Manager: Portage-2.3.89, Repoman-2.3.20
Signed-off-by: Martin Dummer <martin.dummer@gmx.net>

### CODE REVIEW

File: app-admin/passwordsafe/passwordsafe-1.09.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline app-admin/passwordsafe/passwordsafe-1.09.0.ebuild:4]: EAPI-7 should work here.

File: app-admin/passwordsafe/passwordsafe-1.09.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit cmake-utils desktop flag-o-matic wxwidgets
```
Comment (juippis) [inline app-admin/passwordsafe/passwordsafe-1.09.0.ebuild:8]: Use `cmake.eclass` with EAPI-7.

File: app-admin/passwordsafe/passwordsafe-1.09.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+	!minimal? ( !!app-misc/pwsafe )
+	qr? ( media-gfx/qrencode )
+	xml? ( dev-libs/xerces-c )
+	yubikey? ( sys-auth/ykpers )"
+RDEPEND="${COMMON_DEPEND}
+	xvkbd? ( x11-misc/xvkbd )"
+DEPEND="${COMMON_DEPEND}
+	app-arch/zip
+	sys-devel/gettext
+	test? ( dev-cpp/gtest )"
```
Comment (juippis) [inline app-admin/passwordsafe/passwordsafe-1.09.0.ebuild:38]: Note that with EAPI-7 these are all most likely BDEPEND, which also allows you to remove `COMMON_DEPEND` and just inherit `DEPEND` in `RDEPEND`.

File: app-admin/passwordsafe/passwordsafe-1.09.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+S=${WORKDIR}/pwsafe-${MY_PV}
```
Comment (juippis) [inline app-admin/passwordsafe/passwordsafe-1.09.0.ebuild:40]: `S="..."`

File: app-admin/passwordsafe/passwordsafe-1.09.0.ebuild (hunk 1 above)
Comment (madmartin) [reply app-admin/passwordsafe/passwordsafe-1.09.0.ebuild:4]: Hello @juippis ,

I figured out that EAPI-7 does not work without other changes. build fails with a "ld" error. I wanted to find out why in a later step, but with your comments it migth be easier for me now.
Will look at it, thanks for commenting!
Martin

File: app-admin/passwordsafe/passwordsafe-1.09.0.ebuild (hunk 4 above)
Comment (madmartin) [reply app-admin/passwordsafe/passwordsafe-1.09.0.ebuild:40]: @juippis thanks for now, merged your hints.
closing this PR here, new PR for passwordsafe-1.11.0 coming soon

**Pull request closed without merging**